docs/source/reference/api-server/examples/example-deploy-gke-nebius-okta.rst

```diff
@@ -157,7 +157,6 @@ Deploy the API server with helm:
      --set kubernetesCredentials.kubeconfigSecretName=kube-credentials \
      --set gcpCredentials.enabled=true \
      --set gcpCredentials.projectId=$GCP_PROJECT_ID \
-     --set gcpCredentials.serviceAccountJson=$GCP_SERVICE_ACCOUNT_JSON \
```

Comment:
Is this intended?

Reply:
Yes, the `GCP_SERVICE_ACCOUNT_JSON` is used to create the GCP credentials secret, and there is no field `serviceAccountJson` in values.yaml.